shared/templates/selinux_booleans.csv

```diff
@@ -74,13 +74,13 @@ glance_use_fusefs,
 global_ssp,
 gluster_anon_write,
 gluster_export_all_ro,
-gluster_export_all_rw,
+gluster_export_all_rw,disable
 gpg_web_anon_write,
-gssd_read_tmp,
-guest_exec_content,
+gssd_read_tmp,enable
+guest_exec_content,disable
 haproxy_connect_any,
 httpd_anon_write,
-httpd_builtin_scripting,
+httpd_builtin_scripting,disable
```

Comment:
If disabling this, should we add a warning that if the server is using mod_python, mod_perl, mod_php ..., it should be kept enabled (per http://danwalsh.livejournal.com/56760.html?thread=335032 ).